run.py

```diff
@@ -100,7 +100,7 @@ def generate_app(
     synthesis_morphing_parameter = lru_cache(maxsize=4)(_synthesis_morphing_parameter)
 
     # @app.on_event("startup")
-    # async def start_catch_disconnection():
+    # def start_catch_disconnection():
```

Comment:
threading.lockを使うことになったので、この変更は破棄した方が良さそうです。
```suggestion
    # async def start_catch_disconnection():
```

Reply:
あ、多分async消すやつで巻き込んじゃってましたね。ごめんなさい